dev-docs/source/Testing/ErrorReporter-ProjectRecovery/ErrorReporterTesting.rst

```diff
@@ -28,78 +28,110 @@ Error Reporter test
 
 1. Open MantidWorkbench
 
-- Cause a crash by running the `Segfault` algorithm
-- The error report dialog should pop up
+- Cause a crash by running the `Segfault` algorithm, Workbench should close
+- This should cause an error reporter dialog saying Mantid has thrown an unexpected exception
 - Make sure the hyperlink to the privacy policy works
+- Click the `Show More Details` button to open the Show More Details dialog. This should show user details such as OS.
+  The python stacktrace should be empty as the Workbench unhandled exception occurred in C++. Close this dialog.
 - Enter some text in the `Name` box; make sure the `Share non-identifiable information` button gets greyed out
 - In the `Name` box enter `Private` and in the `Email` box enter `private`
-- Click the `Don't share any information` box
-- Check with the database admin
+- Input some additional information into the main textbox. Try to include characters that need to be escaped such as `"`
+- Tick the `Remember Me` checkbox
+- Click the `Don't share any information` button
+- Check with the database admin that an error report was sent withOUT a name, email or stacktrace.
```

Comment:
I think this would be improved by having **without** made bold, rather than capitalising OUT